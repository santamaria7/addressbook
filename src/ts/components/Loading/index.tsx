import React from "react";
import LoadingIcon from "../../../assets/spinner.gif";
const Loading = () => {
    return <div className="loading">
        <img src={LoadingIcon} alt="Data is loading" />
    </div>
};

export default Loading;