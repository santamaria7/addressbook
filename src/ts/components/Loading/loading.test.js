import renderer from "react-test-renderer";
import React from "react";
import Loading from "./index";


describe('user details', function(){
    it('should render the user details with mock data',function(){

        const component = renderer.create(<Loading />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})