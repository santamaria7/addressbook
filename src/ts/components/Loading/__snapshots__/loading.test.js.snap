// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user details should render the user details with mock data 1`] = `
<div
  className="loading"
>
  <div
    className="loadingio-spinner-spinner-fpiuut30cn4"
  >
    <div
      className="ldio-cbw2rtfocq"
    >
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
